rust/feature-flags/src/flag_matching.rs

```diff
@@ -217,9 +217,18 @@ impl FeatureFlagMatcher {
         }
     }
 
-    /// Evaluate feature flags for a given distinct_id
-    /// - Returns a map of feature flag keys to their values
-    /// - If an error occurs while evaluating a flag, it will be logged and the flag will be omitted from the result
+    /// Evaluates all feature flags for the current matcher context.
+    ///
+    /// ## Arguments
+    ///
+    /// * `feature_flags` - The list of feature flags to evaluate.
+    /// * `person_property_overrides` - Any overrides for person properties.
+    /// * `group_property_overrides` - Any overrides for group properties.
+    /// * `hash_key_override` - Optional hash key overrides for experience continuity.
+    ///
+    /// ## Returns
+    ///
+    /// * `FlagsResponse` - The result containing flag evaluations and any errors.
     pub async fn evaluate_all_feature_flags(
```

Comment:
this is an entrypoint method into this struct so added a bit more commentary on it.